ci_build/azure-pipelines/mshost.yaml

```diff
@@ -10,15 +10,22 @@ jobs:
       cd out/Linux
       ./ortcustomops_test
     displayName: Run the native only unit tests
-    # - job: macOS
-#   pool:
-#     vmImage: 'macOS-latest'
-#   steps:
-#   - script: echo hello from macOS
+
+- job: macOS
+  pool:
+    vmImage: 'macOS-latest'
+  steps:
+  - script: |
+      sh ./build.sh -DENABLE_PYTHON=OFF
+    displayName: Build the library and tests
+  - script: |
+      cd out/Darwin
```

Comment:
Does it have a folder called `Darwin`?

Reply:
Darwin is the code name of Apple MacOS, it was generated by 'uname -s'